### ISSUE TITLE
ActionType Type error

### ISSUE THREAD

**Issue by dcantu96** [komarovalexander/ka-table#244]
@komarovalexander hi there, i think I found a type error.

The `ActionType` enum type has a `UpdateTreeGroupsExpanded` with the value being `"UpdateTreeGroupsExpanded "` with an empty space at the end.

```
error TS2551: Property 'UpdateTreeGroupsExpanded ' does not exist on type 'Partial<Record<"ClearFocused" | "ClearSingleAction" | "CloseEditor" | "CloseRowEditors" | "DeleteRow" | "DeselectAllFilteredRows" | "DeselectAllRows" | "DeselectAllVisibleRows" | "DeselectRow" | ... 49 more ... | "Validate", (currentProps: ITableAllProps, params?: any) => void>>'. Did you mean 'UpdateTreeGroupsExpanded'?
```

![Screen Shot 2022-10-03 at 16 37 32](https://user-images.githubusercontent.com/32658690/193689719-981df4fc-a627-4208-b050-45f3e3af8c97.png)

_Originally posted by @dcantu96 in https://github.com/komarovalexander/ka-table/issues/242#issuecomment-1266082317_

**Comment by dcantu96** [comment]
https://github.com/komarovalexander/ka-table/pull/245

**Comment by komarovalexander** [comment]
Thanks for helping!

**Comment by komarovalexander** [comment]
your fix is available in `7.6.3`

**Comment by dcantu96** [comment]
Thanks @komarovalexander ❤️
